cypress/tests/functional/Classic.spec.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * @file cypress/tests/functional/Classic.spec.js
+ *
+ * Copyright (c) 2014-2020 Simon Fraser University
+ * Copyright (c) 2000-2020 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ */
+
+describe('Theme plugin tests', function() {
+	const journalPath = 'publicknowledge';
+	const index = 'index.php';
+	const path = '/' + index + '/' + journalPath;
+	const pages = [
+		'issue/current',
+		'issue/archive',
+		'issue/view/1',
+		'article/view/17',
+		'article/view/17/3',
+		'about',
+		'about/editorialTeam',
+		'about/submissions',
+		'about/contact',
+		'about/privacy',
+		'information/readers',
+		'information/authors',
+		'information/librarians'
+	];
```

Comment:
I recommend putting these directly into `cy.visit()` calls below. When the cypress tests error, you'll get the line of code in the error, but not necessarily the value of the variable. So the error message might not tell you which page failed.